HW1/task1.py

```diff
@@ -0,0 +1,8 @@
+# Задача 1 (2): Найдите сумму цифр трехзначного числа.
+
+print('Введите трехЗначное число: ')
+a = input()
```

Comment:
Эти две строки можно объединить.
```suggestion
a = input('Введите трехЗначное число: ')
```